.version

```diff
@@ -0,0 +1 @@
+1.2
```

Comment:
I guess this file needs to be without the final newline in order not to confuse `cat` or whatever else?

Reply:
How can I exclude it from the check?

Reply:
Did you check this? I just tried it locally. Works even with the final newline.

Reply:
Hmm, it does. That's quite odd. Maybe it would make sense to me in case of `\input` macro but certainly not for `cat`

Reply:
TIL: "Command substitutions strip all trailing newlines from the output of the command inside them"

https://mywiki.wooledge.org/CommandSubstitution

Reply:
TIL too :D